quickwit/quickwit-indexing/src/actors/merge_executor.rs

```diff
@@ -459,7 +459,7 @@ impl MergeExecutor {
                     .expect("A delete task must have a delete query.");
                 let search_request = SearchRequest {
                     index_id: delete_query.index_id,
-                    query: delete_query.query,
+                    query: delete_query.query.map(Into::into),
```

Comment:
Blabla::from would be more readable I think. Here I don't know what the type is.